src/xenia/gpu/vulkan/texture_cache.cc

```diff
@@ -1078,10 +1078,17 @@ bool TextureCache::UploadTexture(VkCommandBuffer command_buffer,
   if (dest->format == VK_FORMAT_D16_UNORM_S8_UINT ||
       dest->format == VK_FORMAT_D24_UNORM_S8_UINT ||
       dest->format == VK_FORMAT_D32_SFLOAT_S8_UINT) {
-    // Do just a depth upload (for now).
-    // This assumes depth buffers don't have mips (hopefully they don't)
-    assert_true(src.mip_levels() == 1);
-    copy_regions[0].imageSubresource.aspectMask = VK_IMAGE_ASPECT_DEPTH_BIT;
+    // Depth buffers can have mips, so dont's assert this:
```

Comment:
If Depth buffers can have mips (hopefully some other dev can validate this), change the line to 
```suggestion
    // Depth buffers can have mips too
```
and remove the commented assert line

Reply:
As explained, kept the original assertion for reference. Not sure how you usually deal with these kind of things for this project.